Examples/Python/femur.py

```diff
@@ -38,7 +38,16 @@ def Run_Pipeline(args):
     if args.interactive:
         input("Press Enter to continue")
 
-    if not args.start_with_prepped_data:
+    if args.skip_grooming:
```

Comment:
If getting the distance transforms file list is the same for all use cases, it is worth modularizing it into a utils function. For example, get_file_list with arguments directory, extension, is_tiny_test, num_samples.